src/extra/buttons.css

```diff
@@ -147,7 +162,16 @@
 @media (--OSdark) {
   :where([type="submit"]),
   :where(form button:not([type="button"])),
-  :where([type="reset"]), 
+  :where([type="reset"]),
+  :where([disabled]) {
+    --_bg: var(--surface-1);
+  }
+}
+
+[data-theme=dark] {
+  :where([type="submit"]),
```

Comment:
```suggestion
  & :where([type="submit"]),
```

syntax requires an `&` to signal the nesting 👍🏻  each of these here need an & in front of them, or chained all within the same `:where()`

Reply:
I figured out why my editor made all these changes! You have trailing spaces all over ;). Any interest in a code formatter to get rid of all these silly nits?

![image](https://user-images.githubusercontent.com/413237/191749170-03d7bd51-27e3-4825-9320-bc95fc6bd142.png)